src/systems/systems/src/air_conditioning/acs_controller.rs

```diff
@@ -427,14 +476,32 @@ impl<const ZONES: usize> ZoneController<ZONES> {
         acs_overhead: &AirConditioningSystemOverhead<ZONES>,
         pack_flow: MassRate,
         pressurization: &impl Cabin,
+        operation_mode: &ACSCActiveComputer,
     ) {
-        self.zone_selected_temperature = acs_overhead.selected_cabin_temperature(self.zone_id);
+        self.zone_selected_temperature = if matches!(operation_mode, ACSCActiveComputer::Secondary)
+        {
+            // If the Zone controller is working on secondary power, the zones are controlled to
+            // 24 degrees by the secondary computer
+            ThermodynamicTemperature::new::<degree_celsius>(24.)
```

Comment:
Not sure how this module could be ported to A380, but having those temperatures as construction parameter could help using this module for either plane (if temperatures are different). If it's totally irrelevant for A380 forget what I just said :)

Reply:
Very good point. I'm just looking into the A380 air con now (which I probably should've done long ago to be honest) and there will be some differences that we'll need to take into account. I will need to go through it in a lot more detail to understand what's common and what isn't though, I suggest leaving it for a future PR :)